content/matplotlib/concepts/pyplot/terms/subplots/subplots.md

```diff
@@ -0,0 +1,62 @@
+| `width_ratios`   | array-like of length ncols, optional                       | Defines the relative widths of the columns. Each column gets a relative width of width_ratios[i] / sum(width_ratios). If not given, all columns will have the same width. Equivalent to gridspec_kw={'width_ratios': [...]}                                                                              |
+| `height_ratios`  | array-like of lenght nrows                                 | Defines the relative heights of the rows. Each row gets a relative height of height_ratios[i] / sum(height_ratios). If not given, all rows will have the same height. Convenience for gridspec_kw={'height_ratios': [...]}.                                                                              |
+| `subplot_kw`     | dict                                                       | Dict with keywords passed to the add_subplot call used to create each subplot.                                                                                                                                                                                                                           |
+| `gridspec_kw`    | dict                                                       | Dict with keywords passed to the GridSpec constructor used to create the grid the subplots are placed on.                                                                                                                                                                                                |
+| `**fig_kw`       |                                                            | All additional keyword arguments are passed to the pyplot.figure call.                                                                                                                                                                                                                                   |
+
+## Example
+In the following example, basic 2 x 2 grid of Axes is formed using **subplots**.
+
+It returs a **Figure** instance and an array of **Axes** objects.
+
+```py
+import matplotlib.pyplot as plt
+import numpy as np
+
+fig, axs = plt.subplots(ncols=2, nrows=2, figsize=(5.5, 3.5),
+                        layout="constrained")
+# add an artist, in this case a nice label in the middle...
+for row in range(2):
+    for col in range(2):
+        axs[row, col].annotate(f'axs[{row}, {col}]', (0.5, 0.5),
+                               transform=axs[row, col].transAxes,
+                               ha='center', va='center', fontsize=18,
+                               color='darkgrey')
+fig.suptitle('plt.subplots()')
+```
+
+![Output of the matplotlib.pyplot.subplots() function](https://matplotlib.org/stable/_images/sphx_glr_arranging_axes_001.png)
```

Comment:
This example should not reproduce an figure on another site, all the elements of an entry should be original content. Update the code and add the relevant image to our docs media folder. See the "file hosting" section of our contribution guide - https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md